### ISSUE TITLE
fix bulleted list

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#422]
![2015-12-24_11-33-30](https://cloud.githubusercontent.com/assets/329735/11996496/30ca95f6-aa32-11e5-9d0d-f98bf88c68b3.png)

Steps to completion list has the number label positioned off center and too small in this box.

Also, we need to look for a jquery plugin (maybe) or some math that suggests when the color needs to be inverted for the text to go from black to white so people can still read it. Right now you could set these to black (for example) and then the number value is hidden completely.

**Comment by btopro** [comment]
![2016-01-04_14-19-05](https://cloud.githubusercontent.com/assets/329735/12098128/2b49e27a-b2ee-11e5-8d1b-d4be0a3bc64d.png)

Now looks like this for the same list style w/ color selection. Commit 1 cleans up spacing / sizing; commit 2 is an automatic complementary color selector. This will be reviewed down the road when we revisit full style guide implementation.
